ansible/inventory.yml

```diff
@@ -14,6 +14,10 @@ hosts:
 
     - 1and1:
         win2008r2-x64-1: {ip: 109.228.48.187}
+      
+    - linaro
+        centos74-armv8-1: {ip: 64.28.99.122}
+        centos74-armv8-2: {ip: 64.28.99.122, port: 2222}
```

Comment:
@bblondin I think this new `port:` attribute needs to be added to the Nagios script, otherwise won't Nagios just complain that there are two machines that have the same IP address?

Reply:
Code changes are required to handle this: https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/204

Reply:
What would happen if this change, adding port, was to be merged? Would something break?